src/main/java/utils/MessageConsole.java

```diff
@@ -0,0 +1,40 @@
+package utils;
```

Comment:
I do not understand the purpose of this component. It seems as though its meant to be tied to the logger/output of experiments. Furthermore, why does a UI element live in the generic Utils package? anything involving swing should realistically live in the frontend package